.eslintrc.js

```diff
@@ -22,6 +22,45 @@ module.exports = {
                 'ghost/ghost-custom/max-api-complexity': 'error'
             }
         },
+        {
+            files: 'core/server/data/migrations/versions/**',
+            excludedFiles: [
+                'core/server/data/migrations/versions/1.*/*',
+                'core/server/data/migrations/versions/2.*/*',
+                'core/server/data/migrations/versions/3.*/*'
+            ],
+            rules: {
+                'ghost/filenames/match-regex': ['error', '^(?:\\d{2}|\\d{4}(?:-\\d{2}){4})(?:-[a-zA-Z]+)+$', true]
+            }
+        },
+        {
+            files: 'core/server/data/migrations/versions/**',
+            rules: {
+                'no-restricted-syntax': ['warn', {
```

Comment:
🤔 does this get correctly merged with the no-restricted syntax errors in the underlying eslint-plugin-ghost without downgrading those errors to warnings?